tools/src/tester/ResultLogger.ts

```diff
@@ -43,7 +44,30 @@ export class ConsoleResultLogger implements ResultLogger {
 
   log_coverage(results: TestResults): void {
     console.log()
-    console.log(`Tested ${results.evaluated_paths_count()}/${results.spec_paths_count()} paths.`)
+    console.log(`Tested ${results.evaluated_paths().length}/${results.spec_paths().length} paths.`)
+  }
+
+  log_coverage_report(results: TestResults): void {
+    console.log()
+    console.log(`${results.unevaluated_paths().length} paths remaining.`)
+    const groups = _.groupBy(results.unevaluated_paths(), (path) => path.split(' ', 2)[1].split('/')[1])
```

Comment:
I was wondering where the space `' '`  character came from because an URL path doesn't contain any space character. Then I saw [this](https://github.com/opensearch-project/opensearch-api-specification/pull/513/files#diff-4535a96ee24bf0e5cabb704065f4bff83e72f485127bcc45184d72b674a7942aR45). The combination of an http verb and an url path constitutes an API operation. I think we should name these entities `unevaluated_operations` and such to avoid confusion.

Reply:
Ok, that's a good suggestion, I'll do a bulk rename.

Reply:
I've refactored the path parsing into an `Operation` object, too.